scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -101,9 +101,9 @@ <h5 class="header">General Institute Settings</h5>
               <!-- Admins can modify loqusdb settings -->
               <div class="col-md-4">
                 {{ form.loqusdb_id.label(class="control-label", data_bs_toggle="tooltip", data_bs_placement="top", title="LoqudDB id" ) }}<br>
-                <select class="selectpicker" name="loqusdb_id" data-style="btn-secondary">
+                <select multiple class="selectpicker" name="loqusdb_id" data-style="btn-secondary">
```

Comment:
👍

Reply:
Is there a difference to `multiple="multiple"`?

Reply:
I didn't know one could specify multiple="multiple". [Here](https://www.w3schools.com/tags/att_select_multiple.asp) they use only "multiple"

Reply:
The important thing is that they all work! :)